migration/v0.32/README.md

```diff
@@ -35,6 +35,40 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
 
 ## Prerequisites
 
+- If `.objectStorage.sync.enabled: true` and `.thanos.objectStorage.type: swift` or `.harbor.persistence.type: swift` then the rclone jobs will automatically use swift for source and destination buckets. You will need to create the application credentials for swift and add them into the `secrets.yaml`:
```

Comment:
It sounded like rclone would use swift for all buckets. Make it clearer that it's just for the ones that are using swift. Maybe something like this:
```suggestion
- If `.objectStorage.sync.enabled: true` and `.thanos.objectStorage.type: swift` or `.harbor.persistence.type: swift` then the rclone jobs will automatically use swift for thanos and/or harbor source and destination buckets. You will need to create the application credentials for swift and add them into the `secrets.yaml`:
```

Reply:
done